packages/wallets/src/features/cashier/modules/TransactionStatus/TransactionStatus.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useCallback } from 'react';
+import { useActiveWalletAccount } from '@deriv/api';
+import { WalletButton, WalletText } from '../../../../components/Base';
+import { Loader } from '../../../../components/Loader';
+import Warning from '../../../../public/images/warning.svg';
+import { CryptoTransaction } from './components/CryptoTransaction';
+import useRecentTransactions from './hooks/useRecentTransactions';
+import './TransactionStatus.scss';
+
+type TTransactionStatus = {
+    transactionType?: 'deposit' | 'withdrawal';
+};
+
+const TransactionStatus = ({ transactionType }: TTransactionStatus) => {
+    const { isError, isLoading, recentTransactions } = useRecentTransactions();
+    const { data: wallet } = useActiveWalletAccount();
+
+    const ErrorState = useCallback(
```

Comment:
Please split this out of the component. If you do conditional render functions, you will get `(React has detected a change in the order of Hooks)`. 

Split out:
- ErrorState
- SuccessState

Reply:
Done